migrations/005_Scores.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TYPE score_status AS ENUM (
+    'failed',
+    'submitted',
+    'best',
+    'mods'
+);
+
+CREATE TABLE scores (
+    id bigserial NOT NULL PRIMARY KEY,
+    beatmap_id int NOT NULL REFERENCES beatmaps (id),
+    user_id int NOT NULL REFERENCES users (id),
+    checksum character(32) NOT NULL,
+    status score_status NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT now(),
+    client_version int NOT NULL,
+    total_score bigint NOT NULL,
+    max_combo int NOT NULL,
+    accuracy real NOT NULL CHECK (accuracy >= 0 AND accuracy <= 100),
+    full_combo boolean NOT NULL,
+    passed boolean NOT NULL,
```

Comment:
I am not sure if we need `passed`, since the `status` already provides that info.

Reply:
I'd say keep